cmd/main.go

```diff
@@ -201,6 +202,21 @@ func maxDuration(dur time.Duration, durations ...time.Duration) time.Duration {
 	return ret
 }
 
+func WrapServeError() func(rw http.ResponseWriter, r *http.Request, err error) {
+	unsupportedHttpCodes := map[int32]struct{}{
+		http.StatusUnprocessableEntity: {},
+	}
+
+	return func(rw http.ResponseWriter, r *http.Request, err error) {
+		if e, ok := err.(errormiddleware.Error); ok {
+			if _, shouldReplace := unsupportedHttpCodes[e.Code()]; shouldReplace {
+				err = errormiddleware.New(http.StatusBadRequest, e.Error())
+			}
+		}
+		errormiddleware.ServeError(rw, r, err)
```

Comment:
I haven't check all our routes, but there is a great chance that won't match our model for all routes
See 
https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L5755
vs
https://github.com/go-openapi/errors/blob/master/api.go#L107-L114

Reply:
Great catch, thanks!